src/app/components/layout/double-button/double-button.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, EventEmitter, Input, OnInit, Output, ViewEncapsulation } from '@angular/core';
+
+export enum DoubleButtonActive {RightButton, LeftButton}
+
```

Comment:
We can keep this component, but this seems like overengineering to me. Let's not create abstract components for design elements that have 3 or less implementations